atd-vze/src/views/Users/Users.js

```diff
@@ -82,16 +101,56 @@ const Users = () => {
             <Col>
               <Card>
                 <CardHeader>
-                  <i className="fa fa-align-justify"></i> Users{" "}
+                  <i className="fa fa-align-justify"></i> Users
                 </CardHeader>
                 <CardBody>
-                  <Row className="align-items-center mb-3">
-                    <Col col="6" sm="4" md="2" xl className="mb-xl-0">
-                      <Link to="/users/add" className="link">
-                        <Button color="primary">
-                          <i className="fa fa-user-plus"></i> Add User
-                        </Button>
-                      </Link>
+                  <Row className="align-items-center">
+                    <Col xs="12" md="6" className="mb-3">
+                      <Row className="d-flex justify-content-md-start justify-content-center ml-0">
```

Comment:
Not a huge deal, but I think that you can get away with removing the `Row` here and below where one is wrapping the `Pagination` component by moving the classes in them to the `Column`s that wrap them.

Reply:
thanks for pointing that out! updated the code to remove those rows and move the styling to the columns